### ISSUE TITLE
Export server not configured

### ISSUE THREAD

**Issue by Pi-George** [highcharts/node-export-server#286]
![image](https://user-images.githubusercontent.com/57261943/122751520-c441da00-d287-11eb-8b0f-8fdcdf369889.png)

Our production build on the 18th suddenly started spitting out this image for every chart. I've been trying to figure out what's changed to cause this but I've had no luck. Running the build.js script and re-installing does nothing to help solve the issue. How can I specifically find out what isn't properly configured so I can try to resolve it?

Line from our dockerfile for installing this repo:

```
    RUN export ACCEPT_HIGHCHARTS_LICENSE=YES && npm config set strict-ssl=false && npm install -g highcharts-export-server --unsafe-perm
```

**Comment by Pi-George** [comment]
Don't know what to make of this, but: ReferenceError: Can't find variable: Intl

```
highcharts-export-server -instr '{}' -outfile mychart.png
    __  ___       __         __               __
   / / / (_)___ _/ /_  _____/ /_  ____ ______/ /______
  / /_/ / / __ `/ __ \/ ___/ __ \/ __ `/ ___/ __/ ___/
 / __  / / /_/ / / / / /__/ / / / /_/ / /  / /_(__  )
/_/ /_/_/\__, /_/ /_/\___/_/ /_/\__,_/_/   \__/____/
        ____//___/                  __     _____
       / ____/  ______  ____  _____/ /_   / ___/___  ______   _____  _____
      / __/ | |/_/ __ \/ __ \/ ___/ __/   \__ \/ _ \/ ___/ | / / _ \/ ___/
     / /____>  </ /_/ / /_/ / /  / /_    ___/ /  __/ /   | |/ /  __/ /
    /_____/_/|_/ .___/\____/_/   \__/   /____/\___/_/    |___/\___/_/
              /_/

                                                                  v2.1.0
Mon Jun 21 2021 16:06:09 GMT+0000 (Coordinated Universal Time) [error] phantom worker 1 unexpected data - ReferenceError: Can't find variable: Intl

  undefined:108
  :83 in P
  :108
  :83
{"filename":"mychart.png"} { data:
   { allowCodeExecution: false,
     width: false,
     callback: false,
     resources: false,
     scale: 1,
     constr: 'Chart',
     chart: { chart: [Object], exporting: [Object] },
     svgstr: undefined,
     format: 'png',
     out: 'mychart.png',
     styledMode: false,
     asyncRendering: undefined,
     async: true,
     reqID: '',
     globalOptions: false,
     themeOptions: false,
     customCode: false,
     dataOptions: false,
     id: 1 },
  working: true,
  ready: false,
  alive: true,
  process:
   ChildProcess {
     _events:
      [Object: null prototype] {
        uncaughtException: [Function],
        error: [Function],
        close: [Function] },
     _eventsCount: 3,
     _maxListeners: undefined,
     _closesNeeded: 3,
     _closesGot: 0,
     connected: false,
     signalCode: null,
     exitCode: null,
     killed: false,
     spawnfile:
      '/usr/lib/node_modules/highcharts-export-server/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs',
     _handle:
      Process { onexit: [Function], pid: 5643, [Symbol(owner)]: [Circular] },
     spawnargs:
      [ '/usr/lib/node_modules/highcharts-export-server/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs',
        '/usr/lib/node_modules/highcharts-export-server/lib/../phantom/worker.js',
        '/usr/lib/node_modules/highcharts-export-server/lib/../' ],
     pid: 5643,
     stdin:
      Socket {
        connecting: false,
        _hadError: false,
        _handle: [Pipe],
        _parent: null,
        _host: null,
        _readableState: [ReadableState],
        readable: false,
        _events: [Object],
        _eventsCount: 1,
        _maxListeners: undefined,
        _writableState: [WritableState],
        writable: true,
        allowHalfOpen: false,
        _sockname: null,
        _pendingData: null,
        _pendingEncoding: '',
        server: null,
        _server: null,
        [Symbol(asyncId)]: 8,
        [Symbol(lastWriteQueueSize)]: 0,
        [Symbol(timeout)]: null,
        [Symbol(kBytesRead)]: 0,
        [Symbol(kBytesWritten)]: 0 },
     stdout:
      Socket {
        connecting: false,
        _hadError: false,
        _handle: [Pipe],
        _parent: null,
        _host: null,
        _readableState: [ReadableState],
        readable: true,
        _events: [Object],
        _eventsCount: 3,
        _maxListeners: undefined,
        _writableState: [WritableState],
        writable: false,
        allowHalfOpen: false,
        _sockname: null,
        _pendingData: null,
        _pendingEncoding: '',
        server: null,
        _server: null,
        [Symbol(asyncId)]: 9,
        [Symbol(lastWriteQueueSize)]: 0,
        [Symbol(timeout)]: null,
        [Symbol(kBytesRead)]: 0,
        [Symbol(kBytesWritten)]: 0 },
     stderr:
      Socket {
        connecting: false,
        _hadError: false,
        _handle: [Pipe],
        _parent: null,
        _host: null,
        _readableState: [ReadableState],
        readable: true,
        _events: [Object],
        _eventsCount: 3,
        _maxListeners: undefined,
        _writableState: [WritableState],
        writable: false,
        allowHalfOpen: false,
        _sockname: null,
        _pendingData: null,
        _pendingEncoding: '',
        server: null,
        _server: null,
        [Symbol(asyncId)]: 10,
        [Symbol(lastWriteQueueSize)]: 0,
        [Symbol(timeout)]: null,
        [Symbol(kBytesRead)]: 0,
        [Symbol(kBytesWritten)]: 0 },
     stdio: [ [Socket], [Socket], [Socket] ] },
  workStartTime: 1624291567055,
  ondone: [Function],
  workcount: 6,
  incoming:
   'ReferenceError: Can\'t find variable: Intl\n\n  undefined:108\n  :83 in P\n  :108\n  :83\n{"filename":"mychart.png"}',
  id: 1,
  doDone: [Function: doDone],
  restart: [Function],
  work: [Function],
  allowRestart: true }
```

**Comment by Pi-George** [comment]
Probably the same issue as https://github.com/highcharts/node-export-server/issues/284

**Comment by KacperMadej** [comment]
Related to https://github.com/highcharts/highcharts/issues/15938
Should be fixed by https://github.com/highcharts/highcharts/pull/15940

**Comment by KacperMadej** [comment]
@udAL 
The related issue was closed and the code was published in version 9.2.0.
Are you able to recreate the missing Intl error using Highcharts in version at least 9.2.0?

**Comment by muttakigenc** [comment]
I have the same. But there is no error in the command line.

~~~
C:\Users\Ömer YILDIZ\Desktop>highcharts-export-server -infile aa.json -outfile chart.png
    __  ___       __         __               __
   / / / (_)___ _/ /_  _____/ /_  ____ ______/ /______
  / /_/ / / __ `/ __ \/ ___/ __ \/ __ `/ ___/ __/ ___/
 / __  / / /_/ / / / / /__/ / / / /_/ / /  / /_(__  )
/_/ /_/_/\__, /_/ /_/\___/_/ /_/\__,_/_/   \__/____/
        ____//___/                  __     _____
       / ____/  ______  ____  _____/ /_   / ___/___  ______   _____  _____
      / __/ | |/_/ __ \/ __ \/ ___/ __/   \__ \/ _ \/ ___/ | / / _ \/ ___/
     / /____>  </ /_/ / /_/ / /  / /_    ___/ /  __/ /   | |/ /  __/ /
    /_____/_/|_/ .___/\____/_/   \__/   /____/\___/_/    |___/\___/_/
              /_/

                                                                  v2.1.0

C:\Users\Ömer YILDIZ\Desktop>
~~~

Can you help me?
